sdk/core/core/inc/_az_cfg.h

```diff
@@ -14,8 +14,9 @@
 
 #ifdef _MSC_VER
 
-// warning C4710: '...': function not inlined
```

Comment:
Is this removed because we are building /W4?

Reply:
To my best understanding, they all do - on my local machine with everything enabled, and in the CI. Thanks for the review Rick!

Reply:
I removed it by commenting out every warning we disable one by one and see it it still builds without warnings. It looks like 2 warnings we had there did no longer apply, because the code has changed over the months.

Reply:
This one wasn't a disable.  It was making it always an error

Reply:
Oops, thank you Rick! Here is the PR to add it back: https://github.com/Azure/azure-sdk-for-c/pull/576/files